### ISSUE TITLE
Greenland is missing

### ISSUE THREAD

**Issue by EightArmCode** [stefangabos/world_countries#82]
Hello!

Many thanks for the repo. 🙏 

I downloaded the English list yesterday and have realized it is missing an entry for Greenland. I do not know about other languages, but I did see that the list in the README is also missing Greenland.

Interesting, because I do see [this](https://github.com/stefangabos/world_countries/blob/c82e357632281a8eb151d2331b3fe5e280f677b8/data/countries/_combined/world.json#L87) in the source.

**Comment by EightArmCode** [comment]
Going through it now, I've actually found quite a few missing countries. Taiwan, Puerto Rico, New Caledonia, etc...

**Comment by stefangabos** [comment]
No countries are missing. As per the long text on the GitHub page, the entries in the `countries` file are, and I quote, 
> the 193 sovereign states (commonly referred to as countries) that are members of the [United Nations (UN)](https://www.un.org/about-us/member-states) plus the 2 [observer states](https://en.wikipedia.org/wiki/United_Nations_General_Assembly_observers) of Palestine and the Vatican City State.

The entries in the `world` file are, and I quote again
> the 249 countries, territories, and areas of geographical interest that have an officially assigned ISO 3166-1 code

Then, it says, and I quote once again :)
> Note that there are 206 sovereign states in the world, the 11 states that are missing from the countries list being the ones having their sovereignty disputed. See the list of [sovereign states](https://en.wikipedia.org/wiki/List_of_sovereign_states) with information on their status and recognition of their sovereignty.

Taiwan falls in the above category as it is not recognized by China, New Caledonia is not technically a country, but is instead recognised as a French overseas territory, Puerto Rico is a territory of the United States, while Greenland is a North American autonomous territory of the Kingdom of Denmark.

Long story short, you want the `world` file not the `countries` one
